### DISCUSSION TITLE
Pin a post in profile

### DISCUSSION THREAD

**Discussion by trueight** [bluesky-social/atproto#2769] [Bluesky Client App]
I want to implement a feature of pin a post on the profile, just like on x, however this is very obvious and someone has probably tried to do it or this is scheduled for a future update, but i searched this in pr's and issues and i not find nothing related, so I want to know if anyone is already working on it or if there something that make this implementation difficult or if I can up a PR with this function.

**Comment by surfdude29** [comment]
One of the devs has actually been working on this today as part of a weekend hack project :)

You can read some of the discussion in this thread:

https://bsky.app/profile/samuel.bsky.team/post/3l2zhgxe4m72b
